Docs/user_guide/modules/terminologies.md

```diff
@@ -0,0 +1,50 @@
+# Terminologies
+
+## Overview
+
+The Terminologies module is a place where the loaded terminology and anatomical contexts can be explored and managed. The same widget (called terminology navigator) appears when editing the "color" of certain types of data nodes (such as models and markups) in the [Data](data.html) module as well as the structures in [Segment Editor](segmenteditor.html), and is used to set the identity and color of the nodes or segments in question.
+Default terminology contexts are the DICOM master list, which is a full list of terminology categories and types, and the 3D Slicer General Anatomy list, which is a subset of the master list, and the entries of which are associated with the labels in the GenericAnatomyColors color table (and the colors matched). Some terminology categories allow the selection of an anatomic region, which come from a different list. There is only one default anatomic context dictionary, which is also called the DICOM master list (but for anatomic contexts). Custom contexts can also be defined.
+The contexts were created as part of the [QIICR](https://github.com/QIICR) project, in the [dcmqi](https://github.com/QIICR/dcmqi) toolkit.
+
+## Use cases
+
+Define terminology and anatomic region of a data node such as model or markup, or an segment in a segmentation, so that it can be identified unambiguously, and associated with standardized codes when exported to DICOM.
```

Comment:
```suggestion
Define terminology and anatomical region of a data node such as model or markup, or a segment in a segmentation, so that it can be identified unambiguously, and associated with standardized codes when exported to DICOM.
```